Mathlib/Combinatorics/Quiver/ConnectedComponent.lean

```diff
@@ -50,18 +57,149 @@ instance [Inhabited V] : Inhabited (WeaklyConnectedComponent V) :=
   ⟨show V from default⟩
 
 protected theorem eq (a b : V) :
-    (a : WeaklyConnectedComponent V) = b ↔ Nonempty (@Path (Symmetrify V) _ a b) :=
+    (a : WeaklyConnectedComponent V) = b ↔
+      Nonempty (@Path (Symmetrify V) _ a b) :=
   Quotient.eq''
 
 end WeaklyConnectedComponent
 
-variable {V}
```

Comment:
I'm guessing you have accidentally removed this line which modifies `V` to be implicit in the definition `wideSubquiverSymmetrify` below: please restore it and the undo the changes to the file ` NielsenSchreier.lean`.

Reply:
(Incidentally if we did wish for `V` to be explicit in the definition `wideSubquiverSymmetrify` then the right fix in `NielsenSchreier.lean` would be to change the expressions like `wideSubquiverSymmetrify T a b` to be `wideSubquiverSymmetrify (Generators G) T a b` or `wideSubquiverSymmetrify _ T a b`. I.e., you would not have to abandon using the `wideSubquiverSymmetrify` definition.)